src/Features/Core/Portable/AddParameter/AbstractAddParameterCodeFixProvider.cs

```diff
@@ -71,15 +78,30 @@ internal abstract class AbstractAddParameterCodeFixProvider<
                 return null;
             }
 
+            if (this.CannotConvertDiagnosticIds.Contains(diagnostic.Id))
+            {
+                return null;
+            }
+
             return initialNode.GetAncestorsOrThis<TArgumentSyntax>()
```

Comment:
what is this trying to do?

Reply:
okay It is trying to get last argument node that its ancestor contains the node if there are multiple argument nodes in the chain like argument of lambda which given as argument of another method call?